accessibilityTest/AccessibilityQuickview.ts

```diff
@@ -38,6 +38,7 @@ export const AccessibilityQuickview = () => {
 
     it('should open an accessible modal', async done => {
       await openQuickview();
+      await Utils.resolveAfter(500);
```

Comment:
I prefer not to hard-code wait-times.

If the async nature of the code is due to a `setTimeout`, it might be possible to overcome using `jasmine.clock()`.
If it is due to a promise, you might be able to overcome it by queuing a promise and awaiting it.

```
await openQuickView()
await Promise.resolve() // add a promise to the end of the queue, and wait for it to resolve.
const axeResults = await axe.run(getModal());
```

Reply:
I'll merge for now. 

This can be revisited if this causes flaky tests.